frontend/src/pages/dataset/components/dataset-tabs/usage-tab/components/curated-queries-list.module.scss

```diff
@@ -0,0 +1,21 @@
+@use '@/styles/globals';
```

Comment:
I think all of this scss can be removed, you can disable Bordered on a list item